common/src/autogluon/common/utils/lite.py

```diff
@@ -0,0 +1,14 @@
+from .utils import get_autogluon_metadata
+
+
+def disable_if_lite_mode(ret=None):
+    def inner(func):
+        def do_nothing(*args, **kwargs):
+            if callable(ret):
+                return ret(*args, **kwargs)
+            return ret
+        metadata = get_autogluon_metadata()
+        if metadata['lite']:
```

Comment:
for better chances at backward compatibility and general robustness, we should first check `if 'lite' in metadata`